db/migrate/20220628163733_convert_activities_to_new_sections_schema.rb

```diff
@@ -0,0 +1,108 @@
+class ConvertActivitiesToNewSectionsSchema < ActiveRecord::Migration
+  class InteractivePage < ActiveRecord::Base
+  end
+
+  class PageItem < ActiveRecord::Base
+  end
+
+  def create_section(page_id, layout, position, show)
+    Section.create(
```

Comment:
Probably this should be added to the list of inline active record objects above like:
```
class Section < ActiveRecord::Base
end
```